Fortinet/fortigate/ingest/parser.yml

```diff
@@ -52,31 +52,31 @@ pipeline:
         timezone: "UTC"
 
   - name: parsed_date
-    filter: '{{parsed_event.message.get("eventtime") != None}}'
+    filter: '{{parsed_event.message.get("timestamp") != None}}'
     external:
       name: date.parse
       properties:
-        input_field: "{{parsed_event.message.eventtime }}"
+        input_field: "{{parsed_event.message.timestamp }}"
         output_field: date
         timezone: "UTC"
```

Comment:
According to some observations:
The timestamp is the local one from the machine.
The eventtime is a timestamp, in nanosecond, in the UTC timezone

So, for the parsing of the timestamp, I suggest using `{{parsed_event.message.tz}}` as the timezone.